### PR TITLE
add tags and misc search options

### PR DESCRIPTION
**Pull request by delphiactual**
add more search options for tags and notes
change 'kinetic' to 'kineticslot' otherwise all stasis items in the kinetic slot are ignored.

### CODE REVIEW

File: src/ts/voiceDim.ts [hunk 1]
```diff
@@ -135,6 +135,13 @@ const otherQueries = {
   'deep site': 'is:deepsight',
   wishlist: 'is:wishlist',
   wishlisted: 'is:wishlist',
+  favorite: 'tag:favorite',
+  keeper: 'tag:keep',
+  junk: 'tag:junk',
+  infusion: 'tag:infuse',
+  archived: 'tag:archive',
+  tagged: 'is:tagged',
+  'with notes': 'is:hasnotes',
```
Comment (mlsof21) [inline src/ts/voiceDim.ts:145]: `with notes` could possibly present issues with how perks are parsed out. I use `with` as a prefix/trigger for looking for perks. I think everything else is ok, though.

Comment (delphiactual) [reply src/ts/voiceDim.ts:145]: ok, I will think about how to word this better

File: src/ts/voiceDim.ts [hunk 2]
```diff
@@ -135,6 +135,15 @@ const otherQueries = {
...
   wishlisted: 'is:wishlist',
+  favorite: 'tag:favorite',
+  keeper: 'tag:keep',
+  junk: 'tag:junk',
+  infusion: 'tag:infuse',
+  archived: 'tag:archive',
+  tagged: 'is:tagged',
+  'not tagged': '-is:tagged',
+  'with notes': 'is:hasnotes',
+  'without notes': '-is:hasnotes',
```
Comment (mlsof21) [inline src/ts/voiceDim.ts:146]: This might actually be ok since the `split` looks for " with " (with spaces). I'd have to test it.

Comment (delphiactual) [reply src/ts/voiceDim.ts]: try it and let me know. I do not have a microphone at my current location.

**Pull request merged**